hypermap/aggregator/tests/mocks/worldmap.harvard.edu/data/search/api

```diff
@@ -907,7 +907,7 @@
          "bbox":{
             "minx":"71.13516205105506",
             "miny":"36.45478671723314",
-            "maxx":"",
+            "maxx":"73.959356679",
```

Comment:
@tomkralidis the mock added in the test suites is based from a real API page. It looks like there are errors from the API here and there. I have noticed several other issues with the API. Anyway the normal case would be to have maxx correctly filled, so it is ok to modify the mock as you did.